packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -183,7 +185,16 @@ const DefaultHeader = ({
                     </div>
                 </div>
             </div>
-            <RealAccountSignup />
+            <RealAccountSignup setShowRiskModal={setShowRiskModal} />
+
+            {
+                show_risk_modal && (
+                    <RiskToleranceWarningModal show_risk_modal={show_risk_modal} setShowRiskModal={setShowRiskModal} />
```

Comment:
Do we need to pass `show_risk_modal` to this component, if it already renders only when show_risk_modal is true?